mParticle-Apple-SDK/Listener/MPListenerController.swift

```diff
@@ -0,0 +1,353 @@
+//
+//  MPListenerController.swift
+//  mParticle-Apple-SDK
+//
+//  Created by Brandon Stalnaker on 12/16/24.
+//
+
+import Foundation
+
+@objc public enum MPEndpoint: Int {
+    case identityLogin = 0, identityLogout, identityIdentify, identityModify, events, config, alias
+}
+@objc public enum MPDatabaseTable: Int {
+    case attributes = 0, breadcrumbs, messages, reporting, sessions, uploads, unknown
```

Comment:
This style is fine for now, but let's discuss later if we want to use this style or enforce a more classic "every item on it's own line" style.